src/main/java/bio/terra/service/duos/DuosDao.java

```diff
@@ -109,21 +118,25 @@ public DuosFirecloudGroupModel retrieveFirecloudGroupByDuosId(String duosId) {
     }
   }
 
-  @Transactional(propagation = Propagation.REQUIRED, isolation = Isolation.SERIALIZABLE)
   public boolean updateFirecloudGroupLastSyncedDate(UUID id, Instant lastSyncedDate) {
-    logger.info("Updating Firecloud group record {} last synced date to {}", id, lastSyncedDate);
+    return updateFirecloudGroupsLastSyncedDate(List.of(id), lastSyncedDate) > 0;
+  }
+
+  @Transactional(propagation = Propagation.REQUIRED, isolation = Isolation.SERIALIZABLE)
+  public int updateFirecloudGroupsLastSyncedDate(List<UUID> ids, Instant lastSyncedDate) {
+    logger.info(
+        "Updating {} Firecloud group record(s) last synced date to {}", ids.size(), lastSyncedDate);
     String sql =
         """
             UPDATE duos_firecloud_group
             SET last_synced_date = :last_synced_date
-            WHERE id = :id
+            WHERE id IN (:ids)
             """;
     MapSqlParameterSource params =
         new MapSqlParameterSource()
-            .addValue("id", id)
+            .addValue("ids", ids)
```

Comment:
Same thing: should we have protection against the case where the id list is empty?

Reply:
Can do (I don't think it does any harm in its current form but we don't need to hit the DB).